<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maindashboard1</class>
 <widget class="QDialog" name="maindashboard1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>761</width>
     <height>761</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(/home/kasturi/carDashboard/img/speedo.jpeg); 
 background-repeat: no-repeat; </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_Km">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:rgb(24, 12, 90);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_Engine">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:rgb(24, 12, 90);</string>
   </property>
  </widget>
  <widget class="QDial" name="dial_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(237, 51, 59) ;</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="leftlabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/img/ll.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rightlabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(/home/kasturi/carDashboard/img/rightarrow.png); </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/img/rr.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="leftbtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>660</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>left indicator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightbtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>660</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>right indicator</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
